Documentation/ceph-quickstart.md

```diff
@@ -39,7 +39,7 @@ cd cluster/examples/kubernetes/ceph
 kubectl create -f operator.yaml
 
 # verify the rook-ceph-operator, rook-ceph-agent, and rook-discover pods are in the `Running` state before proceeding
-kubectl -n rook-ceph-system get pod
+kubectl -n rook-ceph get pod
```

Comment:
@chrisjsimpson In the 0.9 release the operator, agent, and discover pods were actually started by default in the `rook-ceph-system` namespace, so this looks correct already.